build/pipelines/templates/build-single-architecture.yaml

```diff
@@ -1,10 +1,40 @@
-# This template contains steps to build the app for a single architecture.
-# The job containing these steps must set the variables 'BuildConfiguration' and 'BuildPlatform'.
+# This template contains a job to build the app for a single architecture.
 
 parameters:
-  extraMsBuildArgs: ''
+  isReleaseBuild: false
+  platform: ''
+  condition: ''
+
+jobs:
+- job: Build${{ parameters.platform }}
+  displayName: Build ${{ parameters.platform }}
+  condition: ${{ parameters.condition }}
+  pool:
+    vmImage: windows-2019
+  variables:
+    BuildConfiguration: Release
+    BuildPlatform: ${{ parameters.platform }}
+    ${{ if eq(parameters.isReleaseBuild, true) }}:
+      ExtraMSBuildArgs: '/p:IsStoreBuild=true'
+    ${{ if eq(parameters.isReleaseBuild, false) }}:
+      ExtraMSBuildArgs: ''
+  steps:
+  - checkout: self
+    fetchDepth: 1
+
+  - ${{ if eq(parameters.isReleaseBuild, true) }}:
+    - task: UniversalPackages@0
+      displayName: Download internals package
+      inputs:
+        command: download
+        downloadDirectory: $(Build.SourcesDirectory)
+        vstsFeed: WindowsInboxApps
+        vstsFeedPackage: calculator-internals
+        ${{ if eq(parameters.isPublicRelease, true) }}:
```

Comment:
I think `isPublicRelease` can be removed everywhere, since it means the same thing as `isReleaseBuild`.